### PR TITLE
Monthly translation update for OCE (Jan'23)

### PR DESCRIPTION
**Pull request by paulwallace-okta**
@netlify /ja-jp/content/index-admin

### CODE REVIEW

**Review by emikhasyak-okta — APPROVED** [review]
Great job! :ship:

File: ja-jp/Content/Topics/DeploymentGuides/AWS/Notes.htm [hunk 1]
```diff
@@ -344,7 +323,7 @@ <h1>備考</h1>
                                     <!-- *********************** Footer rows ********************************* -->
                                     <div class="footer2" data-test="footer">
                                         <div>
-                                            <p class="copyright" data-test="copyrightNotice">&copy;  <span class="mc-variable okta-variables.Year variable">2022</span> <span class="mc-variable okta-variables.CompanyName variable">Okta, Inc</span>. 全著作権所有。それぞれの商標は、それぞれの商標所有者に帰属します。
+                                            <p class="copyright" data-test="copyrightNotice">&copy;  <span class="mc-variable okta-variables.Year variable">2022</span> <span class="mc-variable okta-variables.CompanyName variable">Okta, Inc</span>. All Rights Reserved. それぞれの商標は、それぞれの商標所有者に帰属します。
```
Comment (mauriciocastillosilva-okta) [inline ja-jp/Content/Topics/DeploymentGuides/AWS/Notes.htm:326]: This is in English. Lets figure out if it is intentional

File: ja-jp/Content/Topics/DeploymentGuides/Sharepoint/Guide/about-people-picker.htm [hunk 2]
```diff
@@ -369,7 +348,7 @@ <h2>重要事項</h2>
                                     <!-- *********************** Footer rows ********************************* -->
                                     <div class="footer2" data-test="footer">
                                         <div>
-                                            <p class="copyright" data-test="copyrightNotice">&copy;  <span class="mc-variable okta-variables.Year variable">2022</span> <span class="mc-variable okta-variables.CompanyName variable">Okta, Inc</span>. 全著作権所有。それぞれの商標は、それぞれの商標所有者に帰属します。
+                                            <p class="copyright" data-test="copyrightNotice">&copy;  <span class="mc-variable okta-variables.Year variable">2022</span> <span class="mc-variable okta-variables.CompanyName variable">Okta, Inc</span>. All Rights Reserved. それぞれの商標は、それぞれの商標所有者に帰属します。
```
Comment (mauriciocastillosilva-okta) [inline ja-jp/Content/Topics/DeploymentGuides/Sharepoint/Guide/about-people-picker.htm:351]: I see in some files the year updated to 2023, but in some files, like this one, it is not. Might be worth investigating why

**Review by mauriciocastillosilva-okta — COMMENTED** [review]
Just a couple of minor comments, looks good overall

File: ja-jp/Content/Topics/DeploymentGuides/Sharepoint/Guide/about-people-picker.htm (hunk 2 above)
Comment (paulwallace-okta) [reply ja-jp/Content/Topics/DeploymentGuides/Sharepoint/Guide/about-people-picker.htm:351]: I grepped, there are 201 files with the old 2022 date.
We had sent this job back with this issue specifically in mind.

**Pull request merged**